.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: "\U0001F41B Bug report"
+about: Report a bug with the specification
+
+---
+
+<!--
+Thank you for reporting a possible bug in ECMA-262.
+
+If this is a bug with a specific implementation of ECMA-262, such as
+a JavaScript engine or a browser, please report it on their bug tracker
+instead.
+-->
+
+**Description:** Adding 1 and 1 is two, but it should be...
+
+**eshost Output:**
+```js
```

Comment:
Instead of a JS code block, I'd make this a full example showing the `eshost` command (so others can easily reproduce/tweak the test):

```
$ eshost -e '1+1'
#### Chakra
2

#### JavaScriptCore
2

#### SpiderMonkey
2

#### V8
2

#### XS
2
```